src/views/docs-view/utils/product-url-adjusters.ts

```diff
@@ -111,6 +111,11 @@ export function rewriteDocsUrl(
 		(basePath: string) => inputUrl.startsWith(`/${basePath}`)
 	)
 	if (isCurrentProductDocsUrl) {
+		// The vagrant vmware utility downloads page is a unique case, where we did
+		// adjust the url structure in the transition to devdot
+		if (currentProduct.slug === 'vagrant' && inputUrl.startsWith('/vmware')) {
+			return `/${currentProduct.slug}/downloads/vmware`
+		}
```

Comment:
are there any other paths under `/vmware/` we need to consider?

Reply:
No just this one downloads page. The vmware documentation lives under [/docs/providers/vmware](https://dev-portal-git-staging-hashicorp.vercel.app/vagrant/docs/providers/vmware)

Reply:
Let me double check though

Reply:
I adjusted it to only rewrite for that specific link. As far as I could see there are no other /vmware pages though